src/wovnio/wovnphp/Headers.php

```diff
@@ -229,20 +238,22 @@ public function browserLang() {
      * @param String $lang The lang to display, can be null or empty
      * @return String The url of the redirections location
      */
-    public function redirectLocation($lang=null) {
-      if ($lang === null)
-        $lang = $this->browserLang();
-      if ($lang === $this->store->settings['default_lang'])
+    public function redirectLocation($lang = null)
+    {
+      if ($lang === null) {
+        $lang = $this->computeBrowserLang();
```

Comment:
where is `computeBrowserLang` defined?

Reply:
Typo, I defined as `computerBrowserLang`, thanks.
It is sad that tests did not catch :'(